airflow/jobs/scheduler_job_runner.py

```diff
@@ -1415,70 +1478,101 @@ def _schedule_dag_run(
         :param dag_run: The DagRun to schedule
         :return: Callback that needs to be executed
         """
-        callback: DagCallbackRequest | None = None
+        trace_id = int(trace_utils.gen_trace_id(dag_run=dag_run), 16)
+        span_id = int(trace_utils.gen_dag_span_id(dag_run=dag_run), 16)
```

Comment:
In addition to another comment where I suggested an "as_int" flag, if you are fetching them both together often, maybe consider a "get_ids" helper to do both and return them as a tuple?  Something like this in the trace_utils module maybe:

```
def get_trace_ids(span_type: str, identifier: str) -> (str, str):
    if span_type == "dag_run:
        span_id = gen_dag_span_id(dag_run=dag_run, as_int=True)
    if span_type == .....
    
    return gen_trace_id(dag_run=dag_run, as_int=True), span_id
 ```

Reply:
Nice idea, but span_id sometimes needs to come from different locations (like task instance, task instance keys, or from dag_run), so this could make the function a bit complicated. I'd like to keep this as is for now.